### PR TITLE
refactor: spore-sdk 0.2.0 migrations

### LINKED ISSUE #49: Outdated SporeScript access in the on-chain-blog tutorial
We used to access a SporeScript directly, like in the reference: 
https://github.com/sporeprotocol/spore-docs/blob/main/docs/tutorials/create-on-chain-blog/create-new-site.mdx#L388

```js
const { script } = predefinedSporeConfigs.Aggron4.scripts.Cluster;
```

In the latest `0.2.0-beta.0` version of the spore-sdk, we can get a Cluster script like this:
```js
const { script } = getSporeScript(predefinedSporeConfigs.Aggron4, 'Cluster');
```

### PR DESCRIPTION
**Pull request by ShookLyngs**
## Description
Adapted spore-sdk 0.2.0 breaking changes:
- `SporeConfig.scripts.xxx` -> `getSporeScript(SporeConfig, 'xxx')`
- Share the testnet SporeConfig globally in the app instead o accessing direcly

## Related issues
- resolves #49

### CODE REVIEW

File: docs/tutorials/create-on-chain-blog/create-new-site.mdx [hunk 1]
```diff
@@ -199,17 +199,33 @@ export async function signTransaction(
 
 This `signTransaction` function streamlines the process of signing transactions. It extracts `signingEntries` from the provided `txSkeleton` and signs the messages within using MetaMask. After signing, it handles the ECDSA recoveryID v parameter and uses `@ckb-lumos/lumos` methods to pack the signature into the Omnilock witness. This process results in the generation of a signed transaction that is ready to send to the Nervos CKB blockchain.
 
-**3. Create Cluster** 
+**3. Share SporeConfig**
```
Comment (anipenguin) [inline docs/tutorials/create-on-chain-blog/create-new-site.mdx:202]: is there an alternative verb for "share" SporeConfig?

Comment (ShookLyngs) [reply docs/tutorials/create-on-chain-blog/create-new-site.mdx:202]: I don't have a better option in mind, how about "use"? Call for help @ashuralyk @Flouse.

Comment (Flouse) [reply docs/tutorials/create-on-chain-blog/create-new-site.mdx:202]: What about `Export SporeConfig`?

`export` is a common concept/keyword in both JS/TS and bash.

**Review by Flouse — APPROVED** [review]

File: docs/tutorials/create-on-chain-blog/create-new-site.mdx (hunk 1 above)
Comment (anipenguin) [reply docs/tutorials/create-on-chain-blog/create-new-site.mdx:202]: export is better than share imo

Comment (ShookLyngs) [reply docs/tutorials/create-on-chain-blog/create-new-site.mdx]: > "export" reads better than "share" imo

Editied

**Review by anipenguin — APPROVED** [review]

**Pull request merged**